crates/store/re_video/src/decode/mod.rs

```diff
@@ -1,4 +1,82 @@
+//!     it also means that we have no chance of displaying the picture perfectly on a screen taking in sRGB (or any other not-matching color space)
+//!   * [Wikipedia says](https://en.wikipedia.org/wiki/Rec._709#Relationship_to_sRGB) sRGB uses the same primaries as BT.709
+//!       * but I also found other sources (e.g. [this forum post](https://forum.doom9.org/showthread.php?p=1640342#post1640342))
+//!         clamining that they're just close enough to be considered the same for practical purposes
+//! * `picture.transfer_characteristics()`
+//!   * until this point everything is "gamma compressed", or more accurately, went through Opto Electric Transfer Function (OETF)
+//!       * i.e. measure of light in, electronic signal out
+//!   * we have to keep in mind the EOTF that our screen at the other end will use which for today's renderpipeline is always sRGB
+//!     (meaning it's a 2.2 gamma curve with a small linear part)
+//!   * Similar to the primaries, BT.709 uses a _similar_ transfer function as sRGB, but not exactly the same
+//!      <https://www.image-engineering.de/library/technotes/714-color-spaces-rec-709-vs-srgb>
+//!        * There's reason to believe players just ignore this:
+//!           * From a [VLC issue](https://code.videolan.org/videolan/vlc/-/issues/26999):
+//!              > We do not support transfers or primaries anyway, so it does not matter
+//!              > (we do support HDR transfer functions PQ and HLG, not SDR ones and we support BT.2020 primaries, but not SMPTE C (which is what BT.601 NTSC is))."
+//!           * …I'm sure I found a report of other video players ignoring this and most of everything except `matrix_coefficients` but I can't find it anymore :(
+//!
+//! All of the above are completely optional for a video to specify and there's sometimes some interplay of relationships with those.
+//! (a standard would often specify several things at once, there's typical and less typical combinations)
+//! So naturally, people will use terms sloppily and interchangeably,
+//! If anything is lacking a video player has to make a guess.
+//! … and as discussed above, even it's there, often video players tend to ignore some settings!
+//!
+//! With all this out of the way…
+//!
+//! ### What's the state of us making use of all these things?
+//!
+//! * ❌ `picture.bit_depth()`
+//!   * TODO(#7594): ignored, we just pretend everything is 8 bits
+//! * ✅ `picture.pixel_layout()`
+//! * ✅ `picture.color_range()`
+//! * 🟧 `picture.matrix_coefficients()`
+//!    * we try to figure out whether to use `BT.709` or `BT.601` coefficients, using other characteristics for guessing if nothing else is available.
+//! * ❌ `picture.primaries()`
+//! * ❌ `picture.transfer_characteristics()`
+//!
+//! We'll very likely be good with this until either we get specific feature requests and/or we'll start
+//! supporting HDR content at which point more properties will be important!
+//!
```

Comment:
This is great 🙏

Reply:
or horrible, depending how you look at it 😄